lib/main.js

```diff
@@ -45,6 +49,36 @@ function addBuiltins(store) {
     'InputVenueMessageContent',
     'InputContactMessageContent',
   ]))
+  store.add(new Union('InputMedia', {
```

Comment:
There wasn't many unions on the page currently, so I figured it would be quicker to get started by defining the remaining two unions. I wanted to get to the feature that provides most value quickly, which is outputting types for methods.

It is definitely possible to automatically add unions, and get rid of these three `store.add(new Union(...` lines. However, that could be done outside of this pull request. Would you be OK with that?

Reply:
OK)))